version-bump.sh

```diff
@@ -61,11 +61,10 @@ else
   echo $BUMP_MODE "version bump detected"
   bump $BUMP_MODE $OLD_VERSION
   echo "pom.xml at" $POMPATH "will be bumped from" $OLD_VERSION "to" $NEW_VERSION
+  cd $POMPATH
```

Comment:
wasnt this the case already. why is this change required

Reply:
this change is required because we want bump only imp core version, so from our workflow we will specify the pom path where we want to bump,
we didn't do it earlier because where we want to bump all pom.xml's in desflow code base, by default it takes as the root directory if we do not mention any pom path